src/main/java/wolox/training/models/Book.java

```diff
@@ -5,6 +5,9 @@
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import org.apache.catalina.User;
```

Comment:
This import is not correct, it is loading a different class that the User class you defined below in this code.